fleche/doc.ml

```diff
@@ -292,7 +289,6 @@ let process_and_parse ~uri ~version ~fb_queue doc last_tok doc_handle =
   let rec stm doc st last_tok =
     let doc = send_eager_diagnostics ~uri ~version ~doc in
     report_progress ~doc last_tok;
-    if Debug.parsing then Io.Log.error "coq" "parsing sentence";
```

Comment:
Why this is gone?

Reply:
There's another line showing the parsed sentence, which I find more useful.

Reply:
There are different concepts, this line is here to let us know parsing has started, and the server is not stuck in some other place.

Reply:
As a general rule I'd suggest we discuss first any further change to the logging infrastructure. I have pretty specific logging needs that can only be discovered if you work in the core engine and all these changes are not suiting my workflow well.